x-pack/plugins/beats_management/public/lib/adapters/framework/kibana_framework_adapter.ts

```diff
@@ -127,20 +127,14 @@ export class KibanaFrameworkAdapter implements FrameworkAdapter {
     }
   }
 
-  public renderUIAtPath(
-    path: string,
-    component: React.ReactElement<any>,
-    toController: 'management' | 'self' = 'self'
-  ) {
+  public renderUIAtPath(path: string, component: React.ReactElement<any>) {
```

Comment:
You removed the additional toController argument, This is needed to ensure I dont have to worry about merge conflicts when I backport changes. I have no current need to have different code in master then in the version branches and would rather not have to.